reference/release-notes/1.21.0.md

```diff
@@ -0,0 +1,70 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.21.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+The 1.21.0 release of Anbox Cloud brings the following features and improvements:
+
+### Core stack improvements
+
+* Initial support for installing Anbox Cloud Appliance on Ubuntu 24.04.<!--AC-2229-->
+* GL Async swap support is disabled by default. See [GL Async swap support](https://discourse.ubuntu.com/t/ams-configuration/20872#gl-async-swap-support-11) for more information.<!--AC-2228-->
+* Production support for Vulkan on NVIDIA GPUs is available:
+  - The default graphics driver stack used is Vulkan, VirGL/Venus and ANGLE. Latency and flickering is improved for VirGL by introducing a dedicated component, frame renderer. <!--AC-2152-->
+  - System tracing is enabled for Android 12 and 13. This helps you to record an Android system trace for identifying ways to improve the performance of your application. See [System tracing overview](https://developer.android.com/topic/performance/tracing) for more information. <!--AC-2161-->
+  - [Renderdoc](https://github.com/baldurk/renderdoc) support for debugging purposes. See [Graphics debugging with renderdoc](tbd) for more information. <!--AC-2093-->
+
+* The NVIDIA UDA driver variant with version 545 or later can be installed with the Anbox Cloud Appliance.<!--AC-2259-->
+* In the [AMS HTTP API](https://anbox-cloud.github.io/latest/ams/), `/1.0` endpoint exposes the cluster ID to enable identifying a subcluster.<!--AC-2148-->
+
```

Comment:
[AC-1700](https://warthogs.atlassian.net/browse/AC-1700) Was missed I think because of the missing label.

Reply:
Added them now